extensions/wikia/Chat2/js/views/views.js

```diff
@@ -412,6 +412,8 @@ var NodeChatDiscussion = Backbone.View.extend({
 });
 //TODO: rename it to frame NodeChatFrame ?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)